app/assets/javascripts/discourse/app/services/lightbox.js

```diff
@@ -251,6 +233,30 @@ export default class LightboxService extends Service {
     );
   }
 
+  handleClickEvent(trigger) {
+    const isLightboxClick = event
```

Comment:
where is event coming from here?

Reply:
Interesting didn't know :+1: but yes I think explicit would be better here.

I was also wondering how that wouldn't be totally broken for you.